### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by SimpleLuke**
- The screenshot will be updated once all styles confirmed
- Please provide me your LinkedIn Links and emails through comments
- Please let me know if any section you want to add in the README
- Please comment on the sentence if you have suggestion about the wordings

To view the README, you may view it on GitHub README-luke branch.

Thank you!

### CODE REVIEW

**Comment by annamaggg** [comment]
linkedin: https://www.linkedin.com/in/anna-magnusson-519658199/ 
email: annamagnusson99@gmail.com

looks good to me, suggestions for line 50: could mention that player can see achievements

**Comment by forreya** [comment]
linkedin: https://www.linkedin.com/in/ryan-lai-a52211214/
email: ryanlai12@gmail.com

**Comment by edwardfgt** [comment]
Linkedin: https://www.linkedin.com/in/edwardthomas99/
email: edwardfgtdev@gmail.com

File: README.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+- Choose your character
+  <img width="590" alt="image" src="https://user-images.githubusercontent.com/89473016/230064312-ff46aaff-bdb0-44d1-8f33-bcf6d3684cca.png">
+
+- Navigate to different features
+  <img width="1433" alt="image" src="https://user-images.githubusercontent.com/89473016/230064676-02ae244d-3788-4133-be11-b8dab07594d1.png">
+
+- Play a Japanese quiz to earn exp and coins through study desk
+  <img width="1437" alt="image" src="https://user-images.githubusercontent.com/89473016/230064788-e0c4766a-ba23-4e65-b664-d4418ebb6b7c.png">
+
+- Buy new outfit with coins through shop
```
Comment (edwardfgt) [inline README.md:164]: Buy new outfits with your coins in the shop

File: README.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  <img width="1433" alt="image" src="https://user-images.githubusercontent.com/89473016/230065058-39a4e89e-7ed8-43da-a621-fe1f92463034.png">
+
+- Change outfit through wardrobe
```
Comment (edwardfgt) [inline README.md:167]: Change outfit in the wardrobe / change outfit through the wardrobe

File: README.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+  <img width="302" alt="image" src="https://user-images.githubusercontent.com/89473016/230066339-4706cb40-d32a-4a64-ac07-ff0d0a2fe277.png">
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ROADMAP -->
+
+## Future roadmap
+
+- [ ] Host the site
+- [ ] Add different categories of quiz
```
Comment (edwardfgt) [inline README.md:186]: Add different categories to the quiz

File: README.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <img width="861" alt="image" src="https://user-images.githubusercontent.com/89473016/230063984-a009feda-367e-413a-b91f-9b39d9175a71.png">
+
+  <h3 align="center">良いね iine</h3>
+
+  <p align="center">
+    A Japanese learning site with a RPG adventure
```
Comment (edwardfgt) [inline README.md:11]: Suggested changes
- A Japanese learning site with an RPG element 
- A Japanese learning site with an RPG twist

**Review by edwardfgt — APPROVED** [review]

**Review by annamaggg — APPROVED** [review]

**Pull request merged**